src/libs/AxelarAssetTransfer.ts

```diff
@@ -158,15 +159,46 @@ export class AxelarAssetTransfer {
     return finalDepositAddress;
   }
 
+  public async getOfflineDepositAddressForERC20Transfer(
+    fromChain: string,
+    toChain: string,
+    destinationAddress: string,
+    fromChainModule: "evm" | "axelarnet" = "evm",
+    tokenSymbol: string,
+    refundAddress?: string
+  ): Promise<string> {
+    await throwIfInvalidChainIds([fromChain, toChain], this.environment);
+    await this.axelarQueryApi.throwIfInactiveChains([fromChain, toChain]);
+
+    refundAddress =
```

Comment:
create a new local binding instead of mutating the parameter, js passes params as ref, not value, meaning this is altering the outer scope